web/src/OverviewResourceBar.tsx

```diff
@@ -231,6 +231,28 @@ function statusCounts(resources: Proto.webviewResource[]): StatusCounts {
   }
 }
 
+function ResourceMetadata(props: { counts: StatusCounts }) {
+  let { total, healthy, pending, unhealthy } = props.counts
+  useEffect(() => {
+    let favicon: any = document.head.querySelector("#favicon")
+    let faviconHref = ""
+    if (unhealthy > 0) {
+      document.title = `Tilt | ❌ (${unhealthy})`
+      faviconHref = "/static/ico/favicon-red.ico"
+    } else if (pending || total === 0) {
+      document.title = `Tilt | ⏲  (${healthy}/${total})`
+      faviconHref = "/static/ico/favicon-gray.ico"
+    } else {
+      document.title = `Tilt | ✅ (${healthy}/${total})`
+      faviconHref = "/static/ico/favicon-green.ico"
```

Comment:
What do you think about this?
- "Tilt" is the most boring bit of info, so we relegate it to the end (which gets truncated if you have a lot of tabs)
- The color of the Favicon is already informative, so the status icon should be grayscale (lest it clash)
- Nix the parens so the important numbers have a better chance of showing up when there are tons of tabs

```suggestion
      document.title = `✖︎ ${unhealthy} ┊ Tilt`
      faviconHref = "/static/ico/favicon-red.ico"
    } else if (pending || total === 0) {
      document.title = `… ${healthy}/${total} ┊ Tilt`
      faviconHref = "/static/ico/favicon-gray.ico"
    } else {
      document.title = `✔︎ ${healthy}/${total} ┊ Tilt`
      faviconHref = "/static/ico/favicon-green.ico"
```

Reply:
love it!